packages/date/src/index.js

```diff
@@ -48,6 +48,19 @@ export const presetValues = [
 	{ value: 'custom', label: __( 'Custom', 'wc-admin' ) },
 ];
 
+export const previousPeriodValues = [
```

Comment:
Nice!  Optional refactor here:  we could use an object here to quickly access all these labels.

```js
export const previousPeriodLabels = [
	today: __( 'Yesterday', 'wc-admin' ),
	yesterday: __( 'Previous Day', 'wc-admin' ),
	week: __( 'Last Week', 'wc-admin' ),
	last_week: __( 'Previous Week', 'wc-admin' ),
	month: __( 'Last Month', 'wc-admin' ),
	last_month: __( 'Previous Month', 'wc-admin' ),
	quarter: __( 'Last Quarter', 'wc-admin' ),
	last_quarter: __( 'Previous Quarter', 'wc-admin' ),
	year: __( 'Previous Quarter', 'wc-admin' ),
	last_year: __( 'Previous Year', 'wc-admin' ),
];
```